spec/helpers/course_path_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe CoursePathHelper do
+  let(:application_choice) { build_stubbed(:application_choice) }
+
+  describe '#offer_path_for' do
+    context 'when :select_option' do
+      it 'returns the application choice path' do
+        expect(helper.course_path_for(application_choice, 'select_option'))
+          .to eq(provider_interface_application_choice_path(application_choice, {}))
+      end
+    end
+
+    context 'when any other step' do
+      it 'returns the step edit path' do
+        expect(helper.course_path_for(application_choice, :other_step))
+          .to eq([:edit, :provider_interface, application_choice, :course, :other_step, params])
```

Comment:
I'm surprised that you could put `params` in the expectation! Could you make it more explicit that the `params` being passed in are the ones expected to come out?